src/swagger-specs/policy-service.yaml

```diff
@@ -1087,185 +1169,252 @@ definitions:
           - DRAFT
           - ENABLED
           - DISABLED
+        example: ENABLED
       marketingActionRefs:
-        description: List of marketing action URI references to which this policy applies
+        description: List of marketing action URI references to which this policy applies.
```

Comment:
Just for consistency, is there a reason we're adding a period here, but not adding a period for something like the property's name just above?